zh/deploy-tidb-binlog.md

```diff
@@ -214,6 +215,8 @@ spec:
 
     `clusterName` 和 `clusterVersion` 必须匹配所需的源 TiDB 集群。
 
+    `initialCommitTs` 为 drainer 没有 checkpoint 时数据同步的起始 commit timestamp，必须以 string 类型配置，如 `"424364429251444742"`。
```

Comment:
Hi @csuzhangxc, for "checkpoint" and "commit timestamp" in the above sentence, would you please write them in Chinese? Thanks!

Reply:
IMO, keep these two words as English is clearer and I think we have used them in Chinese docs.